src/main/resources/application.properties.example

```diff
@@ -0,0 +1,33 @@
+# --- Authorizer configuration
+aserto.authorizer.serviceUrl=localhost:8282
+aserto.authorization.enabled=true
+aserto.authorizer.policyRoot=todoApp
+aserto.authorizer.decision=allowed
+
+## Topaz
+##  This configuration targets a Topaz instance running locally.
+aserto.authorizer.insecure=false
+aserto.authorizer.grpc.caCertPath=${user.home}/.config/topaz/certs/grpc-ca.crt
+
+## Aserto hosted authorizer
+#aserto.tenantId=<tenant_id>
+#aserto.authorizer.policyName=todo
+#aserto.authorizer.policyLabel=todo
+#aserto.authorizer.apiKey=<api_key>
+
+
+
+# --- Directory configuration
+aserto.directory.serviceUrl=localhost:9292
+aserto.directory.insecure=false
+
+# Topaz directory
+aserto.directory.grpc.caCertPath=${user.home}/.config/topaz/certs/grpc-ca.crt
+
+## Aserto hosted directory
+#aserto.directory.apiKey=<api_key>
+#aserto.directory.token=<auth_token>
```

Comment:
Why do we have token and api key for directory?

Reply:
The token should not be there, only the apiKey.

Reply:
Removed `aserto.directory.token`